src/routes.js

```diff
@@ -1,95 +1,129 @@
 // @flow
-import { type Env } from './types';
+import express from 'express';
+import path from 'path';
+import { type LandRequest } from './types';
 import Queue from './Queue';
 import Runner from './Runner';
-import LandRequest from './models/LandRequest';
-import onComment from './events/onComment';
+import Client from './Client';
 import onStatus from './events/onStatus';
+import Logger from './Logger';
 
 function wrap(fn: Function) {
   return (req, res, next) => {
+    Logger.info({ req }, 'Request received');
```

Comment:
tbh, I wasnt sure what this was really doing. Just making each function be a promise?

I'm gonna look at revisiting the logging too. I was thinking of doing something to make it easier to follow the path of a single PR by tagging requests/logs with the commit so that we're able to use that to trace the path from landRequest to merge